spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Placeholder.java

```diff
@@ -28,6 +28,12 @@ class Placeholder {
 
     @NonNull
     String toArgument(List<? extends BugAnnotation> bugAnnotations, @Nullable ClassAnnotation primaryClass) {
-        return bugAnnotations.get(index).format(key, primaryClass);
+        if (index < 0) {
```

Comment:
@gtoison Can you combine the 'if' here?

Reply:
The two look similar but are different (< vs. >), I used the same format as edu.umd.cs.findbugs.FindBugsMessageFormat#format so the same problem will produce a similar output

Reply:
Ah, thanks, I looked at that multiple times but wasn't seeing it.  All good.